include/evmc/helpers.h

```diff
@@ -41,6 +41,21 @@ static inline int evmc_set_option(struct evmc_instance* instance,
     return 0;
 }
 
+/**
+ * Executes the VM.
```

Comment:
Extend documentation to something like `Executes code in the VM`.

Reply:
Done.